rerun_py/rerun_sdk/rerun/archetypes/transform3d_ext.py

```diff
@@ -58,15 +58,19 @@ def __init__(
             if from_parent is None:
                 from_parent = False
 
+            if scale is not None and (not hasattr(scale, "__len__") or len(scale) == 1):  # type: ignore[arg-type]
+                print("reached scalar case")
+                scale = Scale3D(scale)  # type: ignore[arg-type]
+                print(scale)
```

Comment:
```suggestion
```

Reply:
Tip: suffix temporary code with `// TODO: remove` and it won't pass CI

Reply:
argh, good you caught those!